Library/Formula/libsass.rb

```diff
@@ -18,15 +18,11 @@ class Libsass < Formula
 
   def install
     ENV.cxx11
-    ENV["LIBSASS_VERSION"] = "HEAD" if build.head?
+    ENV["LIBSASS_VERSION"] = "3.2.0"
```

Comment:
You need to still hand the `HEAD` case. Is this always needed even for non-HEAD builds now?